.gitignore

```diff
@@ -44,3 +44,15 @@ build/
 ### VS Code ###
 .vscode/
 
+# individual idea files
```

Comment:
You can add "personal files" that change computer to computer into the gitignore if you want - like that xml file, which keeps on changing every merge. 

I predict that if we start using vscode more, we may need to add more things to the Gitignore.